example_2/hardware/include/ros2_control_demo_example_2/visibility_control.h

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017 Open Source Robotics Foundation, Inc.
```

Comment:
Let's change this to this:

```suggestion
// Copyright 2021 ros2_control Development Team
```